### DISCUSSION TITLE
Multiple open content modals using Modals Manager

### DISCUSSION THREAD

**Discussion by lhammarstrom** [mantinedev/mantine#4825] [Feature requests]
First of all: I love Mantine. It's simply the best.

Second: is there a way of keeping multiple open modals using Modals Manager? I would like to be able to open modals on top of each other instead of unmounting the previous opened modals. Looking at the source code I feel like this would be relatively straight forward to implement using something like this:

```
<ModalsContext.Provider value={ctx}>
  {state.modals.map((modal, index) => (
    <Modal
        opened
        zIndex={getDefaultZIndex('modal') + 1 + index}
        {...modal.props}
        onClose={() => closeModal(state.current!.id)}
      >
        {modal.content}
      </Modal>
  ))}

  {children}
</ModalsContext.Provider>
```

I can implement something like this myself, but I'm wondering if there is a way to implement this natively without resorting to importing the Modal component everywhere?

**Comment by rtivital** [comment]
No, modals manager does not support this feature, you can do that only with Modal component

**Reply by atareversei** [reply]
Sorry for pinging this after 2 years, but does Mantine v8 support this feature now?

**Reply by rtivital** [reply]
I do not think so
